examples/analytics.amp.html

```diff
@@ -33,11 +34,22 @@
   "triggers": {
     "defaultPageview": {
       "on": "visible",
-      "request": "event",
+      "request": "pageview",
       "vars": {
         "eventName": "page-loaded",
         "eventId": "42"
       }
+    },
+    "scrollPings": {
+      "on": "scroll",
+      "request": "event",
+      "scroll-spec": {
+        "vertical-boundaries" : [1, 50, 90],
```

Comment:
Can you explain the exact usecase you want? Are you talking about a var to provide a sampled series of scroll positions over time? That would be very independent of this PR.